lib/train/extras/command_wrapper.rb

```diff
@@ -81,6 +81,9 @@ def verify
       when /sudo: sorry, you must have a tty to run sudo/
         ["Sudo requires a TTY. Please see the README on how to configure "\
           "sudo to allow for non-interactive usage.", :sudo_no_tty]
+      when /sudo: a terminal is required to read the password; either use/
+        ["Sudo does not have enough privileges for the operation, "\
+         "try passing password with -P or --ssh-password option", :sudo_lack_of_permission]
```

Comment:
I don't think this text  works coming out of train, with these specific options. This looks more like what we would want Chef Infra Client to be displaying.

Reply:
ohh ok, so  this text will work - `Sudo does not have enough privileges for the operation. Need to pass password` ?

Reply:
Looking closer, I think this is an inaccurate description of the problem. What is the permission that is missing? Isn't this failure about not being able to read from stdin because there's no terminal? 

In that case, maybe something like "Sudo cannot prompt for password because there is no terminal. Please provide the sudo password directly" ?

Reply:
Ok sounds good and apt, i will change the message.

Reply:
I have updated PR with correct error message, please check